quickjs-libc.c

```diff
@@ -2890,6 +2890,57 @@ static int my_execvpe(const char *filename, char **argv, char **envp)
     return -1;
 }
 
+/*
+    This method is called from the child process, after forking
+    and returns the maximum number of open file descriptors on
+    the system.
+ */
+int get_fd_max() {
```

Comment:
Tiny nit: since it doesn't just get max fds but also closes them sometimes, would it make sense to make the function do all the closing (put `closefrom(4)` here too) and rename it to something like `closefds()`?

Reply:
> Tiny nit: since it doesn't just get max fds but also closes them sometimes, would it make sense to make the function do all the closing (put `closefrom(4)` here too) and rename it to something like `closefds()`?

Like this ? (https://github.com/bellard/quickjs/pull/239/commits/697650f1d9dffe0a4755c6a7a2ce33dc2976901c)